src/i18n/Localizable.xcstrings

```diff
@@ -21717,6 +21828,7 @@
       }
     },
     "Share" : {
+      "comment" : "is this one used as gerund or noun?",
```

Comment:
"Share" something with someone, so I guess a verb

Reply:
changed noun to gerund. same reason for"Restart", `because in Korean noun or gerund is usually used to implement actions`